python/ray/_private/gc_collect_manager.py

```diff
@@ -0,0 +1,86 @@
+import gc
+import logging
+import threading
+import time
+from typing import Optional
+
+logger = logging.getLogger(__name__)
+
+_gc_event = threading.Event()
+_gc_thread: Optional[threading.Thread] = None
+# avoid multiple threads triggering GC at the same time
+_gc_lock = threading.Lock()
+
+
+class GCManagerThread(threading.Thread):
```

Comment:
```suggestion
class PythonGCThread(threading.Thread):
```
update the `name` below to match. this will help make it clear to someone who dumps all of the threads in the process what it does